src/pages/hotel/booking/booking-confirmation/user-details/UserDetails.tsx

```diff
@@ -43,7 +57,7 @@ function UserDetails() {
           Mobile Phone <span className="mandatory">*</span>
         </h1>
         <div className="rounded-sm bg-white email border-color">
-          <h1 className="values headings">0033123456789</h1>
+          <h1 className="values headings">{phone}</h1>
```

Comment:
currently user details are hardcoded in service Can you pls replace those hardcoded values with the autopopulated there too